src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -6493,29 +6493,33 @@ void ViewProviderSketch::UpdateSolverInformation()
         std::string msg;
         SketchObject::appendConflictMsg(getSketchObject()->getLastConflicting(), msg);
         signalSetUp(QString::fromLatin1("<font color='red'>%1<a href=\"#conflicting\"><span style=\" text-decoration: underline; color:#0000ff; background-color: #F8F8FF;\">%2</span></a><br/>%3</font><br/>")
-                    .arg(tr("Over-constrained sketch "))
+                    .arg(tr("Over-constrained sketch"))
+                    .arg(" ")

```

Comment:
This isn't right - you can't add more calls to .arg unless they correspond to added placeholders (the `%n` things) in the original string. You need to edit the HTML string above to include the spaces in it (that is, in this case, to add a space after the "%1").

Reply:
Thanks, fixed in 0c99c23